### PR TITLE
Move test-infra-* GitHub teams to sig-testing

### PR DESCRIPTION
**Pull request by spiffxp**

### CODE REVIEW

File: config/kubernetes/sig-testing/teams.yaml [hunk 1]
```diff
@@ -37,3 +37,61 @@ teams:
     - cjwagner
     - mkumatag
     privacy: closed
+  test-infra-admins:
+    description: Administrators to test-infra repo
```
Comment (stevekuznetsov) [inline config/kubernetes/sig-testing/teams.yaml]: Should I be in this one?

File: config/kubernetes/sig-testing/teams.yaml [hunk 2]
```diff
@@ -37,3 +37,61 @@ teams:
...
+    - Random-Liu
+    - rmmh
+    - shyamjvs
+    - stevekuznetsov
+    - sttts
+    - wojtek-t
+    - wonderfly
+    - zmerlynn
+    privacy: closed
+  test-infra-reviewers:
```
Comment (stevekuznetsov) [inline config/kubernetes/sig-testing/teams.yaml]: What's the use of this?

File: config/kubernetes/sig-testing/teams.yaml (hunk 1 above)
Comment (BenTheElder) [reply config/kubernetes/sig-testing/teams.yaml]: this was just a move of the existing lists above to scope them into the SIG. there should be a follow up to clean up the lists.

Comment (BenTheElder) [reply config/kubernetes/sig-testing/teams.yaml]: all of these teams are copied from what is in github, and are therefore almost undoubtedly out of date.

Comment (stevekuznetsov) [reply config/kubernetes/sig-testing/teams.yaml]: SGTM

**Pull request merged**